src/main/java/whackamole/whackamole/Translator.java

```diff
@@ -172,9 +176,13 @@ private Translator(String key, Object... requiredTypes) {
 
 
     private void LookupTranslation() {
+        File langFile = new File(Config.AppConfig.storageFolder + "/locales", Config.AppConfig.Language + ".properties");
+        Properties props = new Properties();
         try {
-            ResourceBundle resource = ResourceBundle.getBundle("Lang", Config.AppConfig.Language);
-            this.value = resource.getString(this.key);
+            FileInputStream getProps = new FileInputStream(langFile);
```

Comment:
Reading the whole file every time is a waste. This should only be done at `onLoad` or `onReload`

Reply:
I stand corrected, this method is already only running on `onLoad` and `onReload`